Packs/CommonTypes/IndicatorFields/indicatorfield-CVSS_Score.json

```diff
@@ -0,0 +1,30 @@
+{
+    "id": "indicator_cvssscore",
+    "version": -1,
+    "modified": "2022-02-15T08:23:28.101948557Z",
+    "name": "CVSS Score",
+    "ownerOnly": false,
+    "cliName": "cvssscore",
+    "type": "number",
+    "closeForm": false,
+    "editForm": true,
+    "required": false,
+    "neverSetAsRequired": false,
+    "isReadOnly": false,
+    "useAsKpi": false,
+    "locked": false,
+    "system": false,
+    "content": true,
+    "group": 2,
+    "hidden": false,
+    "openEnded": false,
+    "associatedTypes": [
+    ],
+    "associatedToAll": true,
+    "unmapped": false,
+    "unsearchable": true,
```

Comment:
You may want to make this seachable. It will fail the SDK validation though so you'll have to ignore that error.

Reply:
I do it this way because of validation :(